metrics-exporter-prometheus/src/exporter/http_listener.rs

```diff
@@ -69,10 +70,12 @@ impl HttpListeningExporter {
         req: &Request<Incoming>,
     ) -> Response<Full<Bytes>> {
         if is_allowed {
-            Response::new(match req.uri().path() {
+            let mut response = Response::new(match req.uri().path() {
                 "/health" => "OK".into(),
                 _ => handle.render().into(),
-            })
+            });
+            response.headers_mut().append(CONTENT_TYPE, HeaderValue::from_static("text/plain"));
```

Comment:
Should this be `text/plain; version=0.0.4`?
https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format